doc/faq/customization.rst

```diff
@@ -85,6 +108,23 @@ A good stating point might be the themes at http://bootswatch.com.
 .. note:: If you add your own CSS file, the file *bootstrap-theme.css* will
    not be loaded anymore. So you might start with a copy of the original file.
 
+
+Use web server substitute module
+................................
+
+You can also use the substitute module of the Apache webserver.
+It is not clear how much performance impact you get, since this
+module can scan and replace all texts that is delivered by the web server.
+
+If you for example want to replace the title of the webpages, you could
+do it like this::
+
+       <Location "/">
+           AddOutputFilterByType SUBSTITUTE text/html
+           Substitute "s/>privacyidea Authentication System</>My own 2FA system</ni"
```

Comment:
Not sure whether the substitute module is case sensitive but the original title is "privacyIDEA Authentication System" (See [`templates/header.html`](https://github.com/privacyidea/privacyidea/blob/0a21fd13a422a45dfe64cafb1f996bff06c188be/privacyidea/static/templates/header.html#L29))

Reply:
The is configured to be case insensitive (the **i** at the end).
Nice as example.
https://httpd.apache.org/docs/2.4/mod/mod_substitute.html#page-header